src/main/java/com/coveo/feign/ReflectionErrorDecoder.java

```diff
@@ -1,5 +1,19 @@
 package com.coveo.feign;
 
+import com.coveo.feign.hierarchy.ClassHierarchySupplier;
```

Comment:
make sure you run `mvn install` in the repo before commiting, there is a format plugin in the lifecycle that will format the source code so that noise like this is avoided.